Documentation/deployment-and-configuration.md

```diff
@@ -40,24 +38,23 @@ fleet's API is served using systemd socket activation.
 At service startup, systemd passes fleet a set of file descriptors, preventing fleet from having to care on which interfaces it's serving the API.
 The configuration of these interfaces is managed through a [systemd socket unit][socket-unit].
 
-[socket-unit]: http://www.freedesktop.org/software/systemd/man/systemd.socket.html
-
-CoreOS ships a socket unit for fleet (`fleet.socket`) which binds to a Unix domain socket, `/var/run/fleet.sock`.
+CoreOS ships a socket unit for fleet (`fleet.socket`) which binds to a Unix domain socket, `/var/run/fleet.sock`. Unix socket is accessible using tool such as curl (v7.40 or greater): `curl --unix-socket /var/run/fleet.sock http:/fleet/v1/units`.
 To serve the fleet API over a network address, simply extend or replace this socket unit.
-For example, writing the following drop-in to `/etc/systemd/system/fleet.socket.d/30-ListenStream.conf` would enable fleet to be reached over the local port `49153` in addition to `/var/run/fleet.sock`:
+For example, writing the following [drop-in] to `/etc/systemd/system/fleet.socket.d/30-ListenStream.conf` would enable fleet to be reached over the local port `49153` in addition to `/var/run/fleet.sock`:
 
 ```
 [Socket]
 ListenStream=127.0.0.1:49153
 ```
 
-After you've written the file, call `systemctl daemon-reload` to load the new drop-in, followed by `systemctl stop fleet.service; systemctl restart fleet.socket; systemctl start fleet.service`.
+After you've written the file, call `systemctl daemon-reload` to load the new [drop-in], followed by `systemctl stop fleet.service; systemctl restart fleet.socket; systemctl start fleet.service`.
 
 Once the socket is running, the fleet API will be available at `http://${ListenStream}/fleet/v1`, where `${ListenStream}` is the value of the `ListenStream` option used in your socket file.
 This endpoint is accessible directly using tools such as curl and wget, or you can use fleetctl like so: `fleetctl --endpoint http://${ListenStream} <command>`.
-For more information, see the [official API documentation][api-doc].
 
-[api-doc]: https://github.com/coreos/fleet/blob/master/Documentation/api-v1.md
+*NOTE:* It is not recommended to listen fleet API TCP socket over public and even private networks. Please use [ssh tunnel][ssh-tunnel] to access remote fleet API.
```

Comment:
Can we rephrase this with out using "note:"? IMO these are the 90% use-cases when you say API, so this is definitely not just a note.

Can we explain _why_ this isn't recommended?